topic_folders/maintainers/email_templates.md

```diff
@@ -1,54 +1,159 @@
+
+In service,
+{name}
+
+#### Available time slots
+ 
+Subject: Schedule for Maintainer Onboarding
+ 
+Dear {name},
+
+Thank you for providing your availability for Maintainer Onboarding. Based on your availability, I have not been able to schedule you into one of the available time slots. The available time slots are shown below. Please let me know if any other these times will work with your schedule, and if so, which you would prefer. I'll then send you a GCal invite and connection information.  
+
+Session A - DATE TIME - DATE TIME UTC (click here for your local time)
+Session B - DATE TIME - DATE TIME UTC (click here for your local time)
+Session C - DATE TIME - DATE TIME UTC (click here for your local time)
+ 
+Thank you again!
+
+In service,
+{name}
+
+
+#### Onboarding reminder
+
+Subject: Reminder email about onboarding schedule
+
+Hi all, 
+
+Just a quick reminder to folks who are signed up for Session A of onboarding that we will be meeting starting at TIME DATE (click this link to see start in your local time). 
+
+You don't need to read anything in preparation for the training, but if you're interested in checking out the onboarding curriculum, you can find it here:
+
+The Zoom link for our training is: LINK. This link should "just work" for you without entering a meeting ID or password.
+
+Please let me know if you have any questions. Looking forward to meeting you all soon!
 
 Best,
 {name}
 
 #### Introduction to co-Maintainers
 
-Subject: Welcome new Maintainer!  
+Subject: Welcome new Maintainer!
 
 CC Current lesson Maintainers
 
-Hi {names},  
+Hi {names},
+
+I would like to introduce {name}, who has just completed Maintainer Onboarding and is now joining the {lesson name} Maintainer team. They are excited to lend a hand on the {lesson name} lesson and to be a part of this maintenance team. Please welcome {name} by introducing yourself and letting {name} know what channel(s) you prefer to be contacted on. 
 
-I would like to introduce {name}, who has just joined the {lesson name} Maintainer team. {name} has experience with {relevant things}. 
-{name} is also {other roles in the community}. They are excited to 
-lend a hand on the {lesson name} lesson and to be a part of this maintenance team. I will be setting {name} up with the appropriate GitHub 
-permissions in the next few days. It would be great if you all could arrange a meeting to discuss logistics and help {name} get oriented.
 Please let me know if there is anything else you all need. Great to have you on the team {name}!
 
+Best, 
+{name}
+
+#### General introduction on Maintainers list
+
+Subject: Welcome new Maintainers!
+ 
+Hi team!
+ 
+Please join me in welcoming NUMBER new Maintainers to our group.
+ 
+NAMES, will be joining on in support of Library Carpentry lessons (LESSON NAMES, respectively). NAME is joining the Software Carpentry LESSON NAME Maintainer team. NAMES will be Data Carpentry lessons (LESSON NAMES, respectively). 
```

Comment:
Is there a reason we phrase things differently for the three lesson programs?

Reply:
I was trying to avoid being too repetitive. But happy to switch to the same wording for each, if you think that would be better. To be honest, I expect I will struggle to come up with another way of phrasing it if we ever gain another Lesson Program 😆

Reply:
@tobyhodges When I read it, the difference in wording to me implies a possible difference in meaning.  I think that repetitive language is OK here.

Reply:
👍 I've adjusted the suggestion above.